crates/bevy_ecs/src/schedule/set.rs

```diff
@@ -30,8 +33,55 @@ pub trait SystemSet: DynHash + Debug + Send + Sync + 'static {
     fn is_anonymous(&self) -> bool {
         false
     }
+
     /// Creates a boxed clone of the label corresponding to this system set.
     fn dyn_clone(&self) -> Box<dyn SystemSet>;
```

Comment:
This method should probably be removed.

Reply:
We cannot remove this, it is used for leaking when `dyn_static_ref` returns `None`.